ext/opentelemetry-exporter-cloud-trace/src/opentelemetry/exporter/cloud_trace/cloud_trace_propagator.py

```diff
@@ -20,13 +20,15 @@
 from opentelemetry.context.context import Context
 from opentelemetry.trace.propagation import httptextformat
 from opentelemetry.trace.span import (
+    INVALID_SPAN_ID,
+    INVALID_TRACE_ID,
     SpanContext,
     TraceFlags,
     get_hexadecimal_trace_id,
 )
 
 _TRACE_CONTEXT_HEADER_NAME = "X-Cloud-Trace-Context"
-_TRACE_CONTEXT_HEADER_FORMAT = r"([0-9a-f]{32})(\/([\d]{0,20}))?(;o=(\d+))?"
+_TRACE_CONTEXT_HEADER_FORMAT = r"^(?P<trace_id>[0-9a-f]{32})\/(?P<span_id>[\d]{20});o=(?P<trace_flags>\d+)$"
```

Comment:
`{20}` should be `{1,20}` I think; this is just an integer string. ..or simply `\d+` since that's what you use for trace_flags.

Reply:
OpenCensus seems to limit span_id length to 20 characters, so I'll do {1,20} here as well.